src/components/SignerPanel/Web3Errors.js

```diff
@@ -88,7 +99,16 @@ export const AccountLocked = ({
       actionText={
         <span>
           Please unlock and{' '}
-          <ButtonLink onClick={onRequestEnable}>enable</ButtonLink>{' '}
+          <ButtonText
+            onClick={onRequestEnable}
+            horizontalPadding="none"
```

Comment:
Maybe we should have `horizontalPadding=none` by default on `ButtonText` :thinking:

Reply:
Or provide an inline mode, since I find myself using it quite a lot in text :).

But yeah, I've almost always had to disable the horizontal padding when I need this component.